react/components/BuyButton/Wrapper.js

```diff
@@ -122,6 +127,13 @@ const BuyButtonWrapper = ({
       ? propDisabled
       : !areAssemblyGroupsValid
 
+  const version =
+    checkoutVersion &&
+    checkoutVersion.installedApp &&
+    checkoutVersion.installedApp.version
+  const checkoutUrl =
+    version && version.split('.')[0] === '1' ? CHECKOUT_URL.V1 : CHECKOUT_URL.V0
```

Comment:
Wouldn't be better apply for any version greater than `0`? It's only require a reorder:

```suggestion
    version && version.split('.')[0] === '0' ? CHECKOUT_URL.V0 : CHECKOUT_URL.V1
```

Reply:
My concern with this change is that if the GraphQL query fails for some reason, it will redirect to the new cart, which might even exist. The way it is written right now at least it will "fallback" to the current cart.

Reply:
Makes sense, in this case what do you think about `parseInt` the value and check for greater than `0`?

```js
  const version =
    checkoutVersion &&
    checkoutVersion.installedApp &&
    checkoutVersion.installedApp.version &&
    parseInt(checkoutVersion.installedApp.version)
  const checkoutUrl =
    version && version.split('.')[0] > 0 ? CHECKOUT_URL.V1 : CHECKOUT_URL.V0
```